usr/src/cmd/dd/dd.c

```diff
@@ -1839,6 +1876,13 @@ static unsigned char
 				"wrote %d bytes, expected %d\n"), bc, oc);
 			term(2);
 		}
+
+		if (ostriden && lseek(obf, (((off_t)ostriden) * ((off_t)obs)),
```

Comment:
Same question about the multiplication. Is there a reason the two conditions are inconsistent for how they check for the stride being enabled?